site/docs/concepts/architecture.md

```diff
@@ -2,27 +2,53 @@
 
 Fybrik takes a modular approach to provide an open platform for controlling
 and securing the use of data across an organization. The figure below showcases the
-current architecture of the Fybrik platform, running on top of Kubernetes. 
+current architecture of the Fybrik platform, running on top of Kubernetes.
 
 The core parts of Fybrik are based on Kubernetes controllers and Custom Resource Definitions (CRDs) in order to define its work items and reconcile the state of them.
 
 The primary interaction object for a data user is the `FybrikApplication` custom resource (1) where a user defines which data should be used for which purpose. The following chart and description describe the architecture and components of Fybrik relative to when they are used.
 
 ![Architecture](../static/architecture.png)
 
-Before the data user can perform any actions a data operator has to [install](../get-started/quickstart.md) Fybrik and modules. 
```

Comment:
I think we need this sentence here and not in the Modules section.

Reply:
Done, it is now just before the Modules section.